packages/core/test/insured-bridge/BridgeAdmin.js

```diff
@@ -395,8 +472,12 @@ describe("BridgeAdmin", () => {
         });
         it("Works with custom gas", async () => {
           const customGasLimit = 10;
-          await bridgeAdmin.methods.setDepositContract(depositBoxImpersonator).send({ from: owner });
-          await bridgeAdmin.methods.setEnableDeposits(l2Token, false, customGasLimit).send({ from: owner });
+          await bridgeAdmin.methods
+            .setDepositContract(chainIds.optimism, depositBoxImpersonator, optimismMessenger.options.address)
+            .send({ from: owner });
+          await bridgeAdmin.methods
+            .setEnableDeposits(chainIds.optimism, l2Token, false, customGasLimit)
+            .send({ from: owner });
           assert.equal(
             l1CrossDomainMessengerMock.smocked.sendMessage.calls[0]._gasLimit,
             customGasLimit,
```

Comment:
do you think it would be worthwhile splitting up these tests into separate files?

it might be nice to have a `BridgeAdmin`, an `Optimism_BridgeAdmin` and `Arbitrum_BridgeAdmin`? otherwise this file will get super long.

also fine to leave this refactor for later.

Reply:
Yeah great point I was being lazy. I built a `MessengerMock` that allows us to split up the `BridgeAdmin` tests from the `OptimismMessenger` logic and used it across all off-chain client testing as well.